plugins/auth/public/auth-oidc/README.md

```diff
@@ -61,12 +61,12 @@ Luigi.setConfig({
 })
 ```
 
-If you want to use the silent token renewal feature, the `silent-callback.html` needs to be copied to a folder in your Luigi Core installation, 
-which is the return path for the IdP provider, configured through the `redirect_uri` setting. The default location of `redirect_uri` is `/assets/auth-oidc/silent-callback.html`. 
+If you want to use the silent token renewal feature, the `silent-callback.html` needs to be copied to a folder in your Luigi Core installation,
+which is the return path for the IdP provider, configured through the `redirect_uri` setting. The default location of `redirect_uri` is `/assets/auth-oidc/silent-callback.html`.
 You must install `oidc-client` in your project as a dev dependency:
 
 ```javascript
-npm i -save-dev copy-webpack-plugin
+npm i -save-dev oidc-client
 ```
 
 Then, you need to import the plugin files and `oidc-client` library in your project using webpack:
```

Comment:
would be great to have a similar explanation regarding the copy-webpack-plugin as in auth/oauth2